### ISSUE TITLE
Bronchocele

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6688]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
A bronchocele is bronchial dilatation due to retained secretions (mucoid impaction) usually caused by proximal obstruction, either congenital (eg, bronchial atresia) or acquired (eg, obstructing cancer). A bronchocele is a tubular or branching Y-or V-shaped structure that may resemble a gloved finger. The CT attenuation of the mucus is generally that of soft tissue but may be modified
by its composition (eg, high-attenuation material in allergic bronchopulmonary aspergillosis). In the case of bronchial atresia, the surrounding lung may be of decreased attenuation because of reduced ventilation and, thus, perfusion.

PMID:18195376

Comment
See Figure 13 of Hansell DM, et al., Fleischner Society: glossary of terms for thoracic imaging. Radiology. 2008 Mar;246(3):697-722.

Parent term (use hpo.jax.org/app)
Abnormal pulmonary thoracic imaging finding HP:0031983
and
Abnormal bronchus morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
